pipeline_dp/__init__.py

```diff
@@ -1,6 +1,8 @@
 from pipeline_dp.budget_accounting import BudgetAccountant
 from pipeline_dp.aggregate_params import AggregateParams
 from pipeline_dp.aggregate_params import Metrics
+from pipeline_dp.aggregate_params import NoiseKind
+from pipeline_dp.dp_computations import *
```

Comment:
Nit: remove `from pipeline_dp.dp_computations import *`

You can use
`from pipeline_dp.dp_computations import *` 
in tests

**Explanation**: __init__.py contains everything that is exposed in the PipelineDP Public API. `dp_computations` are not public API, they are part of implementation. And it's better to decrease number of objects/functions exposed in Public API  - it's less confusing for users and the Public API should be stable (in order not to break user).